pages/src/route/router-service.ts

```diff
@@ -0,0 +1,48 @@
+import {memoize} from '@exadel/esl/src/modules/esl-utils/decorators';
+import {ESLEventUtils} from '@exadel/esl/src/modules/esl-event-listener/core';
+
+export class ESLDemoRouterService {
+  public contentSelector = '#content-routable';
+
+  public previousUrl: URL;
+
+  @memoize()
+  public static init(): ESLDemoRouterService {
+    const router = new ESLDemoRouterService();
+    router.previousUrl = new URL(location.href);
+    ESLEventUtils.subscribe(router, {target: window, event: 'popstate'}, () => ESLDemoRouterService.routeContent(location.pathname, false));
+    return router;
+  }
+
+  public static get instance(): ESLDemoRouterService {
+    return this.init();
+  }
+
+  public static async routeContent(pathname: string, pushState = true): Promise<void> {
+    const router = ESLDemoRouterService.instance;
+    try {
+      const response = await fetch(pathname, {method: 'GET'});
```

Comment:
{method: 'GET'} is default option
```suggestion
      const response = await fetch(pathname);
```